docs/editors-guide/merging-and-obsoleting.md

```diff
@@ -1,17 +1,16 @@
 ## Merging and Obsoleting Classes
 
-### Overview
+## Overview
 
-## When to obsolete / merge
+### When to obsolete / merge
 
-If a term is a candidate for obsoletion and/or merging, this should be reported on the [GitHub issue tracker](https://github.com/monarch-initiative/mondo/issues) and labeled 'obsolete'. Click [here](https://github.com/monarch-initiative/mondo/issues?q=is%3Aissue+is%3Aopen+label%3Aobsolete) for potential terms to be obsoleted.
+If a term is a candidate for obsoletion and/or merging, this should be reported on the <a href="https://github.com/monarch-initiative/mondo/issues" target="_blank">GitHub issue tracker</a> and the issue tagged with the label "obsolete". Click <a href="https://github.com/monarch-initiative/mondo/issues?q=is%3Aissue+is%3Aopen+label%3Aobsolete" target="_blank">here</a> for potential terms to be obsoleted.
 
 Some examples of when to obsolete and/or merge a term are:  
 
-- **Duplicate terms** (for example MONDO:0019055 mitochondrial disease was obsoleted and replaced by MONDO:0004069 'inborn mitochondrial metabolism disorder')  
-- **Out of scope**- Some terms are excluded from Mondo based on certain criteria, which are outlined in the [exclusion reason table](https://mondo.readthedocs.io/en/latest/editors-guide/exclusion-reasons/). For example, terms that are not truly diseases, (ie phenotype terms, such as MONDO:0007348 Colchicine resistance). For another example, see [see #503](https://github.com/monarch-initiative/mondo/issues/503).
-- **obsoleted in source**: for example, OMIM, Orphanet or GARD may retire or obsolete a term. For example, MONDO:0015173 obsolete autoimmune enteropathy type 2
-is a phenotype and not a disease: for example, MONDO:0043606 'obsolete pathologic fracture'
+- **Duplicate terms**: for example "mitochondrial disease" (MONDO:0019055) was obsoleted and replaced by  "inborn mitochondrial metabolism disorder" (MONDO:0004069)
+- **Out of scope**: some terms are excluded from Mondo based on certain criteria, which are outlined in the [exclusion reason table](https://mondo.readthedocs.io/en/latest/editors-guide/exclusion-reasons/). For example, terms that are not truly diseases, (i.e. phenotype terms, such as "Colchicine resistance" (MONDO:0007348). For another example, see [see #503](https://github.com/monarch-initiative/mondo/issues/503).
+- **Obsoleted in source**: external resources such as OMIM, Orphanet, or GARD may retire or obsolete a term. For example "obsolete autoimmune enteropathy type 2" (MONDO:0015173) is a phenotype and not a disease or "obsolete pathologic fracture" (MONDO:0043606)
```

Comment:
(I think I probably wrote this previously) but these examples here are not examples of terms that have been retired in an external source. Here is an example: MONDO:0019348 'obsolete Ehlers-Danlos syndrome with periventricular heterotopia'

Reply:
@nicolevasilevsky can you want to update that in this PR?

Reply:
sure, thanks!

Reply:
I didn't see this one updated so I have now updated it.